src/index.html

```diff
@@ -1,13 +1,428 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Museum</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@700&display=swap"
+    rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@700&family=Raleway:wght@400;500;600;700&display=swap"
+    rel="stylesheet">
+    <link rel="icon" href="/images/favicon-image.png"
+    type="image-favicon">
   </head>
   <body>
-    <h1>Hello Mate Academy</h1>
+    <header class="header">
+      <div class="header__top">
+        <div class="header__top--images">
+          <a href="#">
+            <img class="header__top--image"
+            src="/images/logo.png" alt="logo">
```

Comment:
```suggestion
            <img 
              class="header__top--image"
              src="/images/logo.png" 
              alt="logo"
            >
```
Consider fixing indents according to the code style guide